sky/sky_logging.py

```diff
@@ -143,3 +146,12 @@ def is_silent():
         # threads.
         _logging_config.is_silent = False
     return _logging_config.is_silent
+
+
+def generate_tmp_logging_file_path(file_name: str) -> str:
+    """Generate an absolute path of a tmp file for logging."""
+    run_timestamp = datetime.now().strftime('%Y-%m-%d-%H-%M-%S-%f')
```

Comment:
We should use `backend_utils.get_run_timestamp()` to have paths consistent with provisioning logs.

Reply:
I made some modifications to avoid a circular import.